app/src/main/java/com/mashup/lastgarden/ui/scent/ScentFragment.kt

```diff
@@ -61,19 +63,17 @@ class ScentFragment : BaseViewModelFragment(), ScentViewPagerAdapter.OnClickList
                 Sort.LATEST -> binding.sortButton.text = getString(R.string.radio_btn_middle)
                 else -> binding.sortButton.text = getString(R.string.radio_btn_bottom)
             }
-            //TODO it 번째로 정렬 함수 호출
+            //TODO 필터 API 적용
         }
     }
 
     override fun onCommentClick(scentId: Int) {
-        //TODO id 해당하는 Comment Bottom Sheet 띄우기
-
+        val bottomSheetDialog = ScentCommentBottomSheetFragment()
+        bottomSheetDialog.show(requireActivity().supportFragmentManager, "")
```

Comment:
```suggestion
        ScentCommentBottomSheetFragment().show(requireActivity().supportFragmentManager, "")
```
매개 변수가 없어서 위처럼 하는건 어떤가요!